handlers/join.go

```diff
@@ -24,20 +24,25 @@ func welcomeMessage(channels []Channel) string {
 		welcomeChannels += fmt.Sprintf("- #%s -> %s\n", c.Name, c.Description)
 	}
 
-	return `Welcome to the Gophers Slack channel.
+	return `Welcome to the Gophers Slack community.
+
+*Please read this message; it contains helpful suggestions for interacting within the community.*
+
 This Slack is meant to connect gophers from all over the world in a central place.
 There is also a forum: https://forum.golangbridge.org, you might want to check it out as well.
 We have a few rules that you can see here: http://coc.golangbridge.org.
 
+To share code, you should use: https://play.golang.org/ as it makes it easy for others to help you.
+
+Out of respect for the visually impaired, please refrain from using Slack threads, as they are not accessible for screen readers. Instead, post your replies within the channel.
```

Comment:
```suggestion
Out of respect for those who rely on screen readers to use Slack, we ask that you please refrain from using threads due to accessibility concerns. It's challenging for those users to discover and contribute to active threads in channels, because doing so relies on scrolling up through channel history and visually identifying a threaded conversation. Instead, please post your replies within the channel.
```

Reply:
This is why I'm not a copywriter